dspace-api/src/main/resources/Messages_de.properties

```diff
@@ -0,0 +1,2507 @@
+jsp.layout.footer-default.version-by                            = Erweiterung bereitgestellt und optimiert von
+jsp.layout.header-default.about                                 = \u00DCber tub.dok
+jsp.layout.header-default.alt                                   = tub.dok
+jsp.layout.navbar-admin.authorization                           = Rechte
+jsp.layout.navbar-admin.communities-collections                 = Bereiche/<br />Sammlungen
+jsp.layout.navbar-admin.editlicense                             = Standardlizenz<br/>bearbeiten
+jsp.layout.navbar-admin.editnews                                = Rubriken Aktuelles bearbeiten
+jsp.layout.navbar-admin.epeople                                 = Benutzer
+jsp.layout.navbar-admin.formatregistry                          = Bitstream Format<br />Referenzliste
+jsp.layout.navbar-admin.groups                                  = Gruppen
+jsp.layout.navbar-admin.help                                    = Hilfe
+jsp.layout.navbar-admin.items                                   = Dokumente
+jsp.layout.navbar-admin.logout                                  = Abmelden
+jsp.layout.navbar-admin.metadataimport                          = Metadaten importieren
+jsp.layout.navbar-admin.metadataregistry                        = Metadaten<br/>Referenzliste
+jsp.layout.navbar-admin.statistics                              = Statistik
+jsp.layout.navbar-admin.supervisors                             = Supervisoren
+jsp.layout.navbar-admin.withdrawn                               = Verworfene Dokumente
+jsp.layout.navbar-admin.workflow                                = Gesch\u00E4ftsg\u00E4nge
+jsp.layout.navbar-default.about                                 = \u00DCber tub.dok
+jsp.layout.navbar-default.advanced                              = Erweiterte Suche
+jsp.layout.navbar-default.authors                               = Autor/Autorin
+jsp.layout.navbar-default.browse                                = St\u00F6bern
+jsp.layout.navbar-default.browse-items-by                       = Auflisten nach
+jsp.layout.navbar-default.communities-collections               = Bereiche<br />&amp;&nbsp;Sammlungen
+jsp.layout.navbar-default.date                                  = Datum
+jsp.layout.navbar-default.display-statistics                    = Statistik anzeigen
+jsp.layout.navbar-default.edit                                  = Benutzerprofil bearbeiten
+jsp.layout.navbar-default.go                                    = Los
+jsp.layout.navbar-default.help                                  = Hilfe
+jsp.layout.navbar-default.home                                  = Zur Startseite
+jsp.layout.navbar-default.loggedin                              = Angemeldet&nbsp;&nbsp;als {0}
+jsp.layout.navbar-default.logout                                = Abmelden
+jsp.layout.navbar-default.receive                               = Abonnement<br />Neuerscheinungen
+jsp.layout.navbar-default.search               
```

Comment:
it could be useful to add a comment here explaining that the below message will force the use of the translated label regardless to which value is stored in the database. I guess that most users will be happy to have multilingual label for their cris entities by default but someone could be confused by the change